zk-merge-pr.py

```diff
@@ -34,6 +34,7 @@
 import sys
 import urllib.request, urllib.error, urllib.parse
 import getpass
+import requests
```

Comment:
This can require committers to install extra dependency? Although I may prefer a total builtin solution, `requests` is common and I can live with installing it.